src/Controllers/MoviesController.cs

```diff
@@ -54,5 +53,63 @@ public IActionResult Create(MovieViewModel model)
 
             return View();
         }
+
+        public IActionResult Edit(int? id)
+        {
+            if (id == null)
+            {
+                return NotFound();
+            }
+
+            var movie = _unitOfWork.MovieRepository.Get(id.Value);
+            if (movie == null)
+            {
+                return NotFound();
+            }
+
+            MovieViewModel model = new MovieViewModel();
```

Comment:
another way to do the same in one line and more readable:
`var model = new MovieViewModel { Id = movie.Id, Genre = movie.Genre, Price = movie.Price, ReleaseDate = movie.ReleaseDate, Title = movie.Title }`